src/ivc/cyclefold/sfc/input/assigned.rs

```diff
@@ -660,69 +884,58 @@ impl<const A: usize, F: PrimeField> Input<A, F> {
         );
 
         let mg = MainGate::new(main_gate_config.clone());
-        let is_zero_term = mg.is_zero_term(region, self.step.clone())?;
+        let is_zero_step = mg.is_zero_term(region, self.step.clone())?;
 
         let zero = region.assign_advice(|| "", main_gate_config.state[0], Value::known(F::ZERO))?;
         region.next();
 
-        let expected_l0 = mg.conditional_select(region, &zero, &poly_L_values[0], &is_zero_term)?;
-        let expected_l0 = bn_chip
+        let expected_l0 = mg.conditional_select(region, &zero, &poly_L_values[0], &is_zero_step)?;
+        let expected_l0_limbs = bn_chip
```

Comment:
how is it handled at step=0? both l0 and l1 are zero?

Reply:
Yes, as you can see in conditional select